RxCocoa/RxCocoa/iOS/UITextField+Rx.swift

```diff
@@ -12,21 +12,31 @@ import UIKit
 
 extension UITextField {
     public func rx_text() -> Observable<String> {
+        return observableForControlEvent(.EditingChanged)
+    }
+
+    public func rx_endOnExit() -> Observable<String> {
```

Comment:
Oh :)

The name of the property describes the thing that you are trying to observe. In this case I wanted to observe text, and be notified when text changes. I've implemented it using EditingChanged on osx :) It's just an implementation detail.

When I think about your use case, maybe the correct thing would be just making `observableForControlEvent` public method and renaming it to `rx_textWhen(event: UIControlEvent) -> Observable<String>`. `rx_endOnExit` seems too specific, I don't think we should have a method per event.

You could use that interface like this `rx_textWhen(.EditingChanged)`. Seems readable and simple to me :)

I think that method name should describe thing that you are observing.

When adding methods to extensions (private methods included), one of the reasons why there is rx always in front is to avoid naming collisions in extensions :)

What do you think?